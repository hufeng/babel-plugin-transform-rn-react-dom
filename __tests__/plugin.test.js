const transform = require('../lib');
const babel = require('babel-core');

const example = `
import ReactDOM from 'react-dom';
var ReactDOM1 = require('react-dom');
`;

it('origin code', () => {
  expect(example).toMatchSnapshot();
});

it('match-snapshot', () => {
  const { code } = babel.transform(example, {
    plugins: [transform]
  });

  expect(code).toMatchSnapshot();
});
