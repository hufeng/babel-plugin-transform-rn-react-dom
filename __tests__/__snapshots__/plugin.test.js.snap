// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match-snapshot 1`] = `
"
import ReactDOM from 'react-native';"
`;

exports[`origin code 1`] = `
"
import ReactDOM from 'react-dom';
"
`;
